### ISSUE TITLE
Make the documentation more lightweight and readable

### ISSUE THREAD

**Issue by rluthi** [sfu-db/dataprep#951]
DataPrep is a great tool! And the automatic EDA module is probably the best open-source option out there. 

However, I find your package difficult to understand and learn from your documentation. It makes it feels more complicated than it actually is.

Some ideas to improve :

- Highlight 2 line EDA report creation (only advanced users need the detail)
- Separate documentation from case studies 
- Organise the sub-sections in doc of the Clean module

**Comment by rluthi** [comment]
If you are interested, I could have a go at this

**Comment by anthng** [comment]
> If you are interested, I could have a go at this

I agree with you, the documentation is too difficult to learn. Param in each function does not represent clearly in your docs. I hope Datapre improves to better than, anyway this is the best Auto EDA I've used

**Comment by dovahcrow** [comment]
Yep I agree. We do have an ongoing branch which switches sphinx to jupyterbook and tidies things up a little bit. I'll share the branch here once it can build successfully. Thanks for your interest in helping out! @rluthi
